### DISCUSSION TITLE
Option-Backquote for Toggle Terminal Window

### DISCUSSION THREAD

**Discussion by cubanx** [Eugeny/tabby#9663] [Q&A]
Hello!

Really loving Tabby, and this isn't a make or break thing, but I feel like I'm doing something wrong, so maybe someone can help me.

I'm trying to use `Option-Backquote` as the Quake Mode/Toggle Terminal Window hotkey.

If I set this as the hotkey, nothing happens.

If I use `Ctrl-Backquote` it all works as expected.

Any hints?

Thanks!

**Comment by gugerard** [comment]
Same question here.

I just installed Tabby as a replacement for Guake on Ubuntu 24.04, and I've been loving it so far!
The only missing feature is that I can't use the backquote (²) character as a shortcut instead of Ctrl+Space when I try to define it.

![image](https://github.com/user-attachments/assets/25bd8698-9867-47ce-a279-88d5bc42e363)

Any ideas?
